src/vue/pages/StatisticsGeneral.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div class="statistics-general">
+    <statistics-filters
+      @set-filters-and-update-list="setFiltersAndUpdateList"
+      @show-no-assets-message="showNoAssetsMessage = true"
+    />
+
+    <no-data-message
+      v-if="showNoAssetsMessage"
+      class="statistics-general__no-assets-msg"
+      icon-name="chart-areaspline"
+      :title="'statistics-general.no-assets-title' | globalize"
+      :message="'statistics-general.no-assets-msg' | globalize"
+    />
+
+    <template v-else>
+      <div class="statistics-general__table-wrp">
+        <statistics-general-table
```

Comment:
let `statistics-general-table` just display statistic. transfer to this component skeleton, error and 'no-data-msg'

Reply:
use scheme for displaying all states

Reply:
used